static/app/components/assigneeSelectorDropdown.tsx

```diff
@@ -438,6 +438,18 @@ export default function AssigneeSelectorDropdown({
       }
     }
 
+    // Remove suggested assignees from the member list and team list to avoid duplicates
+    for (let i = 0; i < suggestedAssignees.length; i++) {
```

Comment:
nit: might be a bit simpler to pull up the `suggestedUsers` and `suggestedTeams` and do it this way:

```ts
const suggestedUsers = suggestedAssignees.filter(
  assignee => assignee.type === 'user'
);
const suggestedTeams = suggestedAssignees.filter(
  assignee => assignee.type === 'team'
);

memList = memList.filter(user => !suggestedUsers.find(suggested => suggested.id === user.id))
assignableTeamList = assignableTeamList.filter(team => !suggestedTeams.find(suggested => suggested.id === team.id))
```